corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/form.less

```diff
@@ -208,3 +210,8 @@
     margin-bottom: 0px !important;
   }
 }
+
+div[class*="gr"][class*="col-sm-"][class*="panel"][class*="panel-default"] {
```

Comment:
I'm not familiar with this, what is this targeting? For `gr`, `panel`, and `panel-default`, is there a need to be searching as part of a substring?

Reply:
It needs to search for substring `col-sm-` while others are exact strings. I'm not sure if there is a way to concatenate classes with exact names and a sub-string in a common styling. Therefore, used substring for all.

Updated the image in description.

Reply:
Is the image in the description up to date now? I still see the problem with the collapsible group borders in the description image. I also don't think this targets the right thing. It shouldn't need to target any elements that have only a `col-sm...` class

Reply:
It's updated now in the description. Somehow it was still showing the old image.

> It shouldn't need to target any elements that have only a col-sm... class

It is targeting the `div` that contains the header and body of the group. Since `col-sm-` is quite generic, attributes are applied to combination of all 4 classes in that `div`. Do you have any alternate suggestion?

![image](https://github.com/dimagi/commcare-hq/assets/14318173/afcc1ddf-0e81-4cc7-b08d-1766aa0960ca)

Reply:
Would this work?
```
.question-tile-row {
  .gr {
    padding-right: 0px !important;
    padding-left: 0px !important;
  }
}
```

Reply:
https://github.com/dimagi/commcare-hq/pull/33834/commits/173c955a6bb7cfa5bc3a1cafe54bdca211a32719